docs/user/AnsibleUsage.md

```diff
@@ -0,0 +1,71 @@
+# Using Ansible
+
+## Prereqs
+
+When using the Ansible CLI, make sure you have all the necessary tools [installed on your workstation](Dependencies.md).
+
+## Preparation
+
+### Cloud Authentication
+
+See [ansible cloud authentication](AnsibleCloudAuthentication.md) when deploying to GCP with external postgres
+
+### Variable Definitions (ansible-vars.yaml) File
+
+Prepare your `ansible-vars.yaml` file, as described in [Customize Input Values](../../README.md#customize-input-values). The only variable that cannot be set in the ansible-vars files and instead must be set with the `-e` flag is `-e CONFIG=<path_to_ansible_vars_file>`. All other values can be set either in the ansible-vars.yaml file or using the `-e` extra var param in ansible.
+
+## Running
+
+When running, you need to tell the tool what tasks and actions you would like performed. 
```

Comment:
Declare the Actions and Task to be performed.

Reply:
updated